client-v2/src/components/FrontsEdit/CollectionOverview.tsx

```diff
@@ -69,19 +90,76 @@ const TextRight = styled.div`
 const Name = styled.span`
   color: #333;
   font-weight: 700;
+  padding-right: 0.25em;
 `;
 
+const LastUpdated = styled.span`
+  color: #333;
+  font-weight: 400;
+`;
+
+const StatusFlag = ButtonDefault.extend`
+  outline: transparent;
+  :not(:first-child) {
+    margin-left: 5px;
+  }
+  height: 20px;
+  width: 20px;
+  border-radius: 20px;
+  &:hover,
+  &:active {
+    outline: transparent;
+  }
+`;
+
+const StatusLocked = StatusFlag.extend`
+  &:disabled,
+  &:disabled:hover {
+    cursor: auto;
+  }
+`;
+
+const CollectionStatus = ({
+  hasUnpublishedChanges,
+  hasUnsavedArticleEdits,
+  isUneditable,
+  isBackfilled,
+  lastUpdatedTimeStamp
+}: CollectionStatusProps) => (
+  <>
+    <LastUpdated>{`${distanceFromNow(
+      lastUpdatedTimeStamp as number
+    )} ago`}</LastUpdated>
+
+    {!!isBackfilled ? <LastUpdated> | Backfill</LastUpdated> : null}
```

Comment:
As discussed IRL here you don't need the `!!` throughout here (not that it will change anything but just worthy of note!)